frontend/src/components/content/UserManagementContent.tsx

```diff
@@ -167,21 +167,43 @@ export const UserManagementContent: React.FC<UserManagementContentProps> = ({ us
         }
     };
 
+    const getOrganizationName = (
+        user: IUser | null, 
+        selectedUserType: string, 
+        userVerbose: IUser | null | undefined
+      ): string | undefined => {
+        // Check if the user is a SUPER_ADMIN, ADMIN, or MUNICIPAL_SEG_ADMIN
+        if (user?.userType === USER_TYPES.SUPER_ADMIN || 
+            user?.userType === USER_TYPES.ADMIN || 
+            user?.userType === USER_TYPES.MUNICIPAL_SEG_ADMIN) {
+      
+          // If selected user type is MUNICIPAL or parent is MUNICIPAL_SEG_ADMIN
+          if (selectedUserType === USER_TYPES.MUNICIPAL || 
+              user?.userType === USER_TYPES.MUNICIPAL_SEG_ADMIN) {
+      
+            // Return organization name of the parent user, or fallback to home segment name
+            return userVerbose?.organizationName || userVerbose?.userSegments?.homeSegmentName;
+          }
+        }
+      
+        return undefined; 
+      };
+      
     const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
         event.preventDefault();
         const form = event.target as HTMLFormElement;
         const formData = new FormData(form);
-
+        
+        const orgName = getOrganizationName(user, selectedUserType, userVerbose) ?? 
+                        (selectedUserType === USER_TYPES.BUSINESS || selectedUserType === USER_TYPES.COMMUNITY) ? 
+                        formData.get('inputOrg') as string : undefined;
```

Comment:
Make sure you're using the correct logical operaotrs

Reply:
👌